### ISSUE TITLE
line-per-day report

### ISSUE THREAD

**Issue by unhammer** [bkaestner/org-arbeitszeit#1]
Is this package general enough that one could format a line-per-day report? E.g. from your example org file, I'd like to see something like

```
 | 03/01 | Heading1 | Support Tasks |     8:05 |  485 |
 | 03/01 | Heading1 | Daily Meeting |     0:30 |   30 |
 | 03/01 | Heading2 | Breaks        |     1:05 |   65 |
 | 04/01 | Heading1 | Support Tasks |     8:15 |  495 |
 | 04/01 | Heading1 | Daily Meeting |     0:15 |   15 |
 | 04/01 | Heading2 | Breaks        |     1:10 |   70 |
 |-------+----------+---------------+----------+------|
 |       |          | Sum           | … | … |
```
(if they were all under `* Heading1` or `* Heading2`). 

Or would this be out of the scope for the package? I've been missing a more general clocktable/report interface; manually writing such formatter functions is not very maintainable.

**Comment by bkaestner** [comment]
> Is this package general enough that one could format a line-per-day report?

Hm, that could be provided in a future version, but there's something amiss in your example: you have three lines-per-day at the moment. Your example reminds me a lot more of the usual clocktable, with `:compact t`.

**Comment by unhammer** [comment]
> you have three lines-per-day at the moment

Yes! That's the idea. I changed the title to be more precise :) The `:compact t` table sums per task. 

(I know there is `:step day` but that also includes days that had no clocks on them, and makes a full table per day, even with `:compact t`, not very useful.)

**Comment by bkaestner** [comment]
> but that includes days that had no clocks on them, and makes a full table per day, [...] – so it includes the info I want, but completely drowned in the noise of empty tables and superfluous headings

Aye, I get that. Your table would certainly be interesting as an alternative to the current clocktable, but it's out of scope for `arbeitszeit` unfortunately, as I envision only a single line/entry per time interval (currently fixed to weeks). 

But: everything that's necessary is already in `arbeitszeit` if you want to give it a go. You'd need to use
```lisp
(nth 4 (caddr (org-clock-get-table-data ...)))
```
though and then ensure that you don't accidentally sum up the top-level and lower-level headings together.
